jaeger-core/src/main/java/com/uber/jaeger/senders/UDPSender.java

```diff
@@ -21,31 +21,36 @@
  */
 package com.uber.jaeger.senders;
 
-import com.twitter.zipkin.thriftjava.Span;
+import com.uber.jaeger.Span;
 import com.uber.jaeger.agent.thrift.Agent;
 import com.uber.jaeger.exceptions.SenderException;
+import com.uber.jaeger.reporters.protocols.JaegerThriftSpanConverter;
 import com.uber.jaeger.reporters.protocols.TUDPTransport;
-import org.apache.thrift.TException;
-import org.apache.thrift.protocol.TCompactProtocol;
-import org.apache.thrift.transport.AutoExpandingBufferWriteTransport;
-
+import com.uber.jaeger.thriftjava.Batch;
+import com.uber.jaeger.thriftjava.Process;
 import java.util.ArrayList;
 import java.util.List;
 import lombok.ToString;
+import org.apache.thrift.TBase;
+import org.apache.thrift.TException;
+import org.apache.thrift.protocol.TCompactProtocol;
+import org.apache.thrift.transport.AutoExpandingBufferWriteTransport;
 
 @ToString(exclude = {"spanBuffer", "udpClient", "memoryTransport"})
 public class UDPSender implements Sender {
-  final static int emitZipkinBatchOverhead = 22;
+  final static int emitBatchOverhead = 33;
   private static final String defaultUDPSpanServerHost = "localhost";
-  private static final int defaultUDPSpanServerPort = 5775;
+  private static final int defaultUDPSpanServerPort = 6832;
```

Comment:
Interesting... https://github.com/uber/jaeger/blob/master/cmd/agent/app/config.go#L127 shows that port 6832 is for the jaeger-binary protocol but we're using compactProtocol here. Not sure how the endtoend tests are passing.